app/src/main/java/org/glucosio/android/tools/SplitDateTime.java

```diff
@@ -20,44 +20,42 @@
 
 package org.glucosio.android.tools;
 
-import org.joda.time.DateTime;
-import org.joda.time.format.DateTimeFormat;
-import org.joda.time.format.DateTimeFormatter;
-
 import java.text.DateFormat;
+import java.text.SimpleDateFormat;
 import java.util.Date;
 
 public class SplitDateTime {
 
-    private DateTime origDateTime;
+    private Date origDateTime; // Example "yyyy-MM-dd HH:mm"
     private DateFormat inputFormat;
 
-    public SplitDateTime(Date origDatetime) {
-        this.origDateTime = new DateTime(origDatetime);
+    public SplitDateTime(Date origDatetime, DateFormat origDateFormat) {
+        this.origDateTime = origDatetime;
+        this.inputFormat = origDateFormat;
     }
 
     public String getHour() {
-        DateTimeFormatter hourFormatter = DateTimeFormat.forPattern("HH");
```

Comment:
I reverted the joda.time uses, because it retrieved time -2 hours :/
Perhaps we could create another issue to take care of joda.time in another Pull request ?